src/data/list/infix.lean

```diff
@@ -307,6 +329,24 @@ begin
   exact infix_append _ _ _
 end
 
+instance : is_partial_order (list α) (<+:) :=
+{ refl := prefix_refl,
+  trans := λ _ _ _, is_prefix.trans,
+  antisymm := λ l₁ l₂ h₁ h₂,
+    eq_of_prefix_of_length_eq h₁ (le_antisymm (is_prefix.length_le h₁) (is_prefix.length_le h₂)) }
```

Comment:
Does this work with dot notation as
```suggestion
  antisymm := λ l₁ l₂ h₁ h₂, eq_of_prefix_of_length_eq h₁ $ h₁.length_le.antisymm h₂.length_le }
```
If so, can you change the other proofs too?